main/client/header/navbar.js

```diff
@@ -0,0 +1,11 @@
+Meteor.startup(function() {
+
+  // Adds class .shrinked to #top-nav
+  $(window).scroll(function() {
+    let topNav = $('nav#top-nav');
+    if ($(document).scrollTop() > 75) topNav.addClass('shrinked');
```

Comment:
Add comments to these lines, describing their purpose in plain English. This supports us when we review and revise the code.